Makefile

```diff
@@ -30,11 +30,7 @@ runmigrations:
 
 runlint:
 	$(PYLINT) wellregistry/registry
-	$(PYLINT) wellregistry/registry/migrations/0001_registry_table.py
-	$(PYLINT) wellregistry/registry/migrations/0002_add_agency_groups.py.py
-	$(PYLINT) wellregistry/registry/migrations/0003_country_lookups.py
-	$(PYLINT) wellregistry/registry/migrations/0004_lookup_tables.py
-	$(PYLINT) wellregistry/registry/migrations/0005_lookups_fk.py
+	$(PYLINT) wellregistry/registry/migrations/0002_add_agency_groups.py
```

Comment:
I removed the auto generated ones from linting. We don't need to lint these and they are automatically removed via the django-pylint plugin.